packages/amplify_core/example/ios/Podfile

```diff
@@ -0,0 +1,41 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
```

Comment:
If we want this to run I think we might need to uncomment this line and set to `:ios, 11.0`

Reply:
This example is unused.